cluster-api-gcp/deploy/deploy_helper.go

```diff
@@ -293,3 +299,21 @@ func (d *deployer) waitForApiserver(master string, timeout time.Duration) error
 	}
 	return err
 }
+
+// Make sure the default service account in kube-system namespace exists.
+func (d *deployer) waitForServiceAccount(timeout time.Duration) error {
```

Comment:
Do we follow the same pattern as the other retries? Namely, using attempts + sleep? See: https://github.com/karan/kube-deploy/blob/760c8e423e826f61afb7b7748bdb1761c8a30bed/cluster-api-gcp/deploy/deploy_helper.go#L38

Reply:
It doesn't seem like the linked const is used anywhere. I'm using the retry logic implemented in `waitForApiserver` (right above this new method). Ideally, we should have a common retry annotation/helper/whatever the go equivalent is.

Reply:
Got it. Feel free to do it in another PR (file an issue to track comment in that case).

Reply:
Done. https://github.com/kubernetes/kube-deploy/issues/562

Thanks for the review